_sections/intro.md

```diff
@@ -16,4 +16,4 @@ In the 2020 Dev Days, we hope to **increase the awareness of Nilearn's developme
 We also hope to **highlight future directions** where we are especially excited to see **community-driven development** efforts.
 
 
-
+> the event does not require registration to listen
```

Comment:
Light suggestion to specify that we would prefer if folks register for the sprint, but not the talks !

```suggestion
The talks for the Software Development and Scientific days will be freely available with public links shared on the event schedule.
We strongly encourage registration for the Code Sprint, which can be done [here](/dev-days-2020/sprint.html)
```

Reply:
I would recommend highlighting this point, in a big way. a lot of people seem to be asking about this.